src/screens/Episode/helpers/getTime.ts

```diff
@@ -0,0 +1,29 @@
+import * as dateFns from 'date-fns'
+import { UNIX_TIME_START } from '../common'
+import { DateFormatType, getFormattedDate } from 'src/helpers'
+
+const checkHourExist = (hours: string): boolean => {
+  return hours === '00' ? false : true
+}
+
+const getTime = (secondsAsNumber: number): string => {
+  const timeInFormatHHmmss = getFormattedDate(
+    String(dateFns.addSeconds(new Date(UNIX_TIME_START), secondsAsNumber)),
+    DateFormatType.HOURS_MINUTES_SECONDS
+  )
+
+  const splittedTime = timeInFormatHHmmss.split(':')
+  const hours = splittedTime[0]
+  const minutes = splittedTime[1]
+  const seconds = splittedTime[2]
```

Comment:
What does it mean `1`, `2`, `3`?
Let's use [destructuring assignment](https://javascript.info/destructuring-assignment)
```suggestion
  const [hours, minutes, seconds] = splittedTime
```